### PR TITLE
String manipulation 3

### PR DESCRIPTION
**Pull request by Lee-SungWon**
Screen Shot 2020-09-12 at 10 11 18 AM

### CODE REVIEW

File: string-manipulation-3/src/is-anagram.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  for (let i = 0; i < secondStringNoSpace.length; i++) {
+    if (firstStringNoSpace.includes(secondStringNoSpace[i])) {
+      counter2++;
+    }
+  }
+
+  if (counter === firstStringNoSpace.length && counter2 === secondStringNoSpace.length && firstStringNoSpace.length === secondStringNoSpace.length) {
+    return true;
+  } return false;
```
Comment (thebearingedge) [inline string-manipulation-3/src/is-anagram.js]: If you remove the spaces and sort the characters, the strings should equal each other.

File: string-manipulation-3/src/title-case.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      if (newArr[j].includes(':')) {
+        word = 'JavaScript:';
+      } else {
+        word = 'JavaScript';
+      }
+      newArr[j] = word;
+      // const splice1 = newArr.splice(0, j);
+      // splice1.push(word);
+      // const splice2 = newArr.splice(1, newArr.length - 1);
+      // newArr = splice1.concat(splice2);
```
Comment (thebearingedge) [inline string-manipulation-3/src/title-case.js]: Please do not turn in commented out code

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, just some minor clean up and one of your algorithms can be simplified as well.

File: string-manipulation-3/src/title-case.js (hunk 2 above)
Comment (Lee-SungWon) [reply string-manipulation-3/src/title-case.js]: revised!

File: string-manipulation-3/src/is-anagram.js (hunk 1 above)
Comment (Lee-SungWon) [reply string-manipulation-3/src/is-anagram.js]: revised!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**